### PR TITLE
Third tutorial about workpiece geometry

### PR DESCRIPTION
**Pull request by vhirtham**
## Changes

Adds a new tutorial about workpiece geometry

- [ ] Link first tutorial
- [ ] Add to "Getting started"
- [ ] Fix todos

### CODE REVIEW

File: tutorials/01_03_geometry.ipynb [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "geometry.to_file(\n",
+    "    \"weldx_geometry.stl\", profile_raster_width=Q_(\"2mm\"), trace_raster_width=Q_(\"50mm\")\n",
+    ")"
+   ]
```
Comment (CagtayFabry) [inline tutorials/01_03_geometry.ipynb:349]: I get the following `RuntimeWarning` in the last cell (STL export):
```python
...\site-packages\meshio\stl\_stl.py:185: RuntimeWarning: invalid value encountered in true_divide
  normals = (normals.T / nrm).T
```
Is this expected?

**Review by CagtayFabry — CHANGES_REQUESTED** [review]
please also include a `k3d` plot in this tutorial

File: tutorials/01_03_geometry.ipynb [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3d plot (matplotlib)\n",
+    "\n",
+    "We can also create a nice 3d plot from the groove and the seam length stored in the WelDX file.\n",
+    "All we need fore this is the `Geometry` class of WelDX.\n",
```
Comment (CagtayFabry) [inline tutorials/01_03_geometry.ipynb:143]:
```suggestion
    "All we need for this is the `Geometry` class of WelDX.\n",
```

File: tutorials/01_03_geometry.ipynb [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The 3d plot you see does not show a closed surface but is composed of multiple triangles.\n",
+    "The reason for this is that the default rendering backend, matplotlib, wasn't meant to be a 3d plotting tool.\n",
+    "Therefore, it lacks certain features that would be necessary for a correct plot of a closed 3d surface.\n",
+    "Because all those plotted lines can make it sometimes hard to identify the actual shape, the plot function gives you the choice to plot only the control points instead of the triangles.\n",
+    "This can be achieved by setting the `show_wireframe` parameter to `False`:"
```
[the remark this answers is not in the document]
Comment (vhirtham) [reply tutorials/01_03_geometry.ipynb:194]: I have rewritten this specific cell

File: tutorials/01_03_geometry.ipynb (hunk 1 above)
Comment (vhirtham) [reply tutorials/01_03_geometry.ipynb:349]: Not nice, but expected. This error simply says that we have one or more degenerated triangles (no surface cause all points lie on a straight line) in the mesh. They are located at the top and the bottom surfaces and result from the simplified triangulation algorithm we are using for those surfaces. They have no visual impact so you won't see it in the plots. I have removed the warning in the tutorial by simply changing the rasterization parameters but a real fix would require us to use a proper triangulation algorithm or to filter the degenerated triangles.

Comment (CagtayFabry) [reply tutorials/01_03_geometry.ipynb:349]: ok, if it is as simple as choosing better rasterization parameters then that should work 👍

**Pull request closed without merging**